api/ccapi/include/model.h

```diff
@@ -141,12 +141,16 @@ class Model {
   /**
    * @brief     Run Model training and validation
    * @param[in] values hyper parameters
+   * @param[in] stop_cb callback function to decide stop training or not
```

Comment:
Describe the semantics of stop_cb's return and input argument here.

Reply:
Added